hyp3_insar_gamma/ifm_sentinel.py

```diff
@@ -345,46 +321,44 @@ def gammaProcess(masterFile, slaveFile, outdir, dem=None, dem_source=None, rlook
     else:
         logging.info("Found azimuth offset of {}!".format(offset))
 
-    output = masterDateShort + "_" + slaveDateShort
+    output = reference_date_short + "_" + secondary_date_short
 
     process_log("Starting s1_coreg_overlap")
     execute(f"S1_coreg_overlap SLC1_tab SLC2R_tab {output} {output}.off.it {output}.off.it.corrected",
             uselogging=True, logfile=log)
 
     process_log("Starting interf_pwr_s1_lt_tops_proc.py 2")
-    interf_pwr_s1_lt_tops_proc(master, slave, hgt, rlooks=rlooks, alooks=alooks, step=3)
+    interf_pwr_s1_lt_tops_proc(reference, secondary, hgt, rlooks=rlooks, alooks=alooks, step=3)
 
     # Perform phase unwrapping and geocoding of results
     process_log("Starting phase unwrapping and geocoding")
-    unwrapping_geocoding(master, slave, step="man", rlooks=rlooks, alooks=alooks)
+    unwrapping_geocoding(reference, secondary, step="man", rlooks=rlooks, alooks=alooks)
 
     #  Generate metadata
     process_log("Collecting metadata and output files")
 
-    execute(f"base_init {master}.slc.par {slave}.slc.par - - base > baseline.log",
+    execute(f"base_init {reference}.slc.par {secondary}.slc.par - - base > baseline.log",
             uselogging=True, logfile=log)
     os.chdir(wrk)
 
     etc_dir = os.path.abspath(os.path.dirname(hyp3_insar_gamma.etc.__file__))
     shutil.copy(os.path.join(etc_dir, "sentinel_xml.xsl"), ".")
 
-    execute(f"xsltproc --stringparam path {masterFile} --stringparam timestamp timestring"
-            f" --stringparam file_size 1000 --stringparam server stuff --output {master}.xml"
-            f" sentinel_xml.xsl {masterFile}/manifest.safe", uselogging=True, logfile=log)
+    execute(f"xsltproc --stringparam path {reference_file} --stringparam timestamp timestring"
+            f" --stringparam file_size 1000 --stringparam server stuff --output {reference}.xml"
+            f" sentinel_xml.xsl {reference_file}/manifest.safe", uselogging=True, logfile=log)
 
-    execute(f"xsltproc --stringparam path {slaveFile} --stringparam timestamp timestring"
-            f" --stringparam file_size 1000 --stringparam server stuff --output {slave}.xml"
-            f" sentinel_xml.xsl {slaveFile}/manifest.safe", uselogging=True, logfile=log)
-
-    makeHDF5List(master, slave, outdir, output, dem_source, logname)
+    execute(f"xsltproc --stringparam path {secondary_file} --stringparam timestamp timestring"
+            f" --stringparam file_size 1000 --stringparam server stuff --output {secondary}.xml"
+            f" sentinel_xml.xsl {secondary_file}/manifest.safe", uselogging=True, logfile=log)
```

Comment:
I think it's possible everything here can go, other than `os.chdir(wrk)`.  I don't see us including *any* xml, metadata, or log files in the output package other than the readme.  The hdf5/xsltproc business mirrors the old workflow in hyp3-rtc-gamma for building the .iso.xml file; hdf5.txt would have been passed to the MapReady write_hdf5_xml command.

If that's the case, we can also drop `etc/sentinel_xml.xsl`.

Maybe that should all wait for a separate PR, though.

Reply:
also lets us drop `xsltproc` from the Dockerfile

Reply:
Opened issue #73